### ISSUE TITLE
[Legitimate Site Blocked] opunsia.com

### ISSUE THREAD

**Issue by 1gbk** [MetaMask/eth-phishing-detect#12347]
https://opunsia.com/ Is legit website for e-commerce store, please whitelist.

Thank you.

**Comment by AlexHerman1** [comment]
hi there,

we are not unblocking sites that are not hosting anything. you can open a new PR when you have something stood up and can provide documentation on it.
